src/sql/workbench/contrib/welcome/page/browser/welcomePage.ts

```diff
@@ -228,6 +227,7 @@ class WelcomePage extends Disposable {
 		@IHostService private readonly hostService: IHostService,
 		@IFileService fileService: IFileService,
 		@IProductService private readonly productService: IProductService,
+		@ICommandService private readonly commandService: ICommandService,
```

Comment:
I would add this to the end of the parameter list instead of inserting it to a random place.